packages/utils/src/eslint-utils/applyDefault.ts

```diff
@@ -25,7 +25,6 @@ function applyDefault<TUser extends readonly unknown[], TDefault extends TUser>(
   (options as unknown[]).forEach((opt: unknown, i: number) => {
     if (userOptions[i] !== undefined) {
       const userOpt = userOptions[i];
-
```

Comment:
[Nitpick] Unrelated change, let's take it out. No need to increase the diff size.